src/main/resources/db/migration/all/20240716095118__remove_national_ap_area.sql

```diff
@@ -0,0 +1 @@
+DELETE FROM ap_areas where identifier = 'National';
```

Comment:
I love it when a plan comes together 😎

Reply:
So much work had to go into enabling that one little line. Thanks for your help working all of this out!